src/Core/Types/PlatformPropertyType.cs

```diff
@@ -49,8 +50,8 @@ protected override string GetNameInProperFormat(string name, int id)
         {
             StringBuilder sb = new StringBuilder(name);
 
-            sb.Replace("Platform_", "");
-            sb.Replace("PropertyId", "");
+            sb.Replace(DisplayStrings.Platform_, "");
+            sb.Replace(DisplayStrings.PropertyId, "");
```

Comment:
These are used for mapping to a code construct. I would not recommend moving them into a resource file